packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -85,15 +101,79 @@ const defaultTreeConfigurationOptions: Required<ITreeConfigurationOptions> = {
 	enableSchemaValidation: false,
 };
 
+/**
+ * Property-bag configuration for {@link TreeViewConfiguration} construction.
+ * @public
+ */
+export interface ITreeViewConfiguration<TSchema extends ImplicitFieldSchema = ImplicitFieldSchema> {
+	/**
+	 * The schema which the application wants to view the tree with.
+	 */
+	readonly schema: TSchema;
+
+	/**
+	 * If `true`, the tree will validate new content against its stored schema at insertion time
+	 * and throw an error if the new content doesn't match the expected schema.
+	 *
+	 * @defaultValue `false`.
+	 *
+	 * @remarks Enabling schema validation has a performance penalty when inserting new content into the tree because
+	 * additional checks are done. Enable this option only in scenarios where you are ok with that operation being a
+	 * bit slower.
+	 */
+	readonly enableSchemaValidation?: boolean;
+}
+
+/**
+ * Configuration for {@link ITree.viewWith}.
+ * @public
+ */
+export class TreeViewConfiguration<TSchema extends ImplicitFieldSchema = ImplicitFieldSchema> {
+	/**
+	 * The schema which the application wants to view the tree with.
+	 */
+	public readonly schema: TSchema;
+
+	/**
+	 * If `true`, the tree will validate new content against its stored schema at insertion time
+	 * and throw an error if the new content doesn't match the expected schema.
+	 *
+	 * @defaultValue `false`.
+	 *
+	 * @remarks Enabling schema validation has a performance penalty when inserting new content into the tree because
+	 * additional checks are done. Enable this option only in scenarios where you are ok with that operation being a
+	 * bit slower.
+	 */
+	public readonly enableSchemaValidation: boolean;
```

Comment:
These are duplicated in ITreeViewConfiguration.

I think it might be good to add `implements Required<ITreeViewConfiguration<TSchema>>` to TreeViewConfiguration to associate these in the type system and intelisense.

For actual generated docs, you can use inheritDoc to deduplicate the doc comments.